saw-script/src/SAWScript/Interpreter.hs

```diff
@@ -977,8 +1052,29 @@ interpretTopStmt printBinds stmt = do
          setCryptolEnv cenv'
          --showCryptolEnv
 
+    SS.StmtInclude pos _file ->
+      panic "interpretTopStmt" [
+          "Leftover unresolved include statement",
+          "Position: " <> Text.pack (show pos)
+      ]
+
     SS.StmtTypedef _ _ name ty ->
       liftTopLevel $ addTypedef name ty
+    SS.StmtPushdir _ dir -> liftTopLevel $ pushdir dir
+    SS.StmtPopdir _ -> liftTopLevel popdir
+
+-- | Interpret multiple top-level statements in an interpreter monad
+--   (any of the SAWScript monads)
+--
+--    This is the entry point used by the REPL for executing stuff the
+--    user types in.
```

Comment:
Minor style nit pick: weird indentation here. Consider aligning all of these the same:


```suggestion
-- | Interpret multiple top-level statements in an interpreter monad
--   (any of the SAWScript monads)
--
--   This is the entry point used by the REPL for executing stuff the
--   user types in.
```